src/PresentationalComponents/Cards/OverviewDashbarCard/__tests__/OverviewDashbarCard.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import { OverviewDashbarCard } from '../OverviewDashbarCard';
+import {
+  PATHWAYS,
+  INCIDENTS,
+  CRITICAL_RECOMMENDATIONS,
+  IMPORTANT_RECOMMENDATIONS,
+} from '../../../../AppConstants';
+
+// this component returns the appropriate Card component based on the title it receives
+describe('OverviewDashbarCard', () => {
+  const user = userEvent.setup();
+
+  it('Should render Pathways Card', async () => {
```

Comment:
You might be able to combine these tests, as they seem to be the same or similar. See https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
Thank you for the comment ! 
I have added what you suggested in the new PR 👍🏼 
Here is a link to the file in the new PR: 
https://github.com/RedHatInsights/insights-advisor-frontend/pull/1205/files#diff-c795923a4523816bd39cfff57d24127022c53fb5afc86c35cae5169280a244cd